en-US/rooms/premium-twin.html

```diff
@@ -24,7 +24,7 @@ <h6>Room type</h6>
           <p>Twin</p>
           <h6>Occupancy</h6>
           <p>3 persons</p>
-          <h6>Surface Area</h6>
```

Comment:
I heard surface area would include the area of the floor, walls, ceiling, etc. 😅